editor/components/button/style.scss

```diff
@@ -1,8 +1,14 @@
 .editor-button {
 	background: none;
 	border: none;
+	outline: none;
 
 	&:disabled {
 		opacity: 0.6;
 	}
+
+	&.is-active {
```

Comment:
I don't know? :) I've always been confused with the `:active` state. I think `.is-active` and `:active` are different states, maybe we should rename `.is-active`?